tests/link/numba/test_elemwise.py

```diff
@@ -117,6 +117,25 @@ def test_Elemwise(inputs, input_vals, output_fn, exc):
         compare_numba_and_py(out_fg, input_vals)
 
 
+def test_elemwise_speed(benchmark):
```

Comment:
Can you add a case with broadcasting?

Reply:
That one is using broadcasting, isn't it?

Reply:
Hmm I guess I wanted non-dimshuffled broadcasting because that's the one we've been thinking about, but should be the same?

Reply:
Elemwise will always call dimshuffle in make_node anyway, so there shouldn't be a non-dimshuffled case?

Reply:
By non-dimshuffled I mean matrix + row/col